### PR TITLE
feat: cc-0007 Show PriceBar on Blank Page, Add open close button

### PR DESCRIPTION
**Pull request by qiying417**
Add open close button to the pricebar to match the UI design.

image

### CODE REVIEW

**Review by JasonCaoAU — APPROVED** [review]
LGTM

File: src/components/PriceBar/index.tsx [hunk 1]
```diff
@@ -43,6 +45,9 @@ const PriceBar: React.FC = () => {
   return (
     <>
       <Container>
+        <OpenCloseButton>
+          <AiOutlineDown />
+        </OpenCloseButton>
```
Comment (tatoMa) [inline src/components/PriceBar/index.tsx:50]: Is this for card cc007 or cc008? If for cc008, the button should open and close the price bar as I read from our notion board.
The solution can be applied by using a state conditionally render the price bar.

**Review by tatoMa — COMMENTED** [review]
Also maybe move the testing files into the components folder.

File: src/components/PriceBar/index.tsx (hunk 1 above)
Comment (qiying417) [reply src/components/PriceBar/index.tsx:50]: This is for cd007 AC1, "I can see the price bar with “v” on the top." Just for UI purpose.
The open close button feature which open/close the price bar will be implemented in cc008.

**Review by tatoMa — APPROVED** [review]

File: src/components/PriceBar/index.tsx [hunk 2]
```diff
@@ -62,7 +67,7 @@ const PriceBar: React.FC = () => {
             <FormLabel htmlFor="ruler-on" mb={0}>
               Ruler On
             </FormLabel>
-            <Switch id="ruler-on" />
+            <Switch id="ruler-on" colorScheme={"blue"}/>
```
Comment (devharris7) [inline src/components/PriceBar/index.tsx:70]: remove `{}`

**Review by devharris7 — COMMENTED** [review]
add a single

**Pull request merged**